app/views/static_pages/index.html.haml

```diff
@@ -7,11 +7,11 @@
 
 %section.container#BestSeller
     %h1 Best Selling Products
-    %div.gridContainer
-        %div.gridContent
-            %p Best Seller n.1
-
-        %div.gridContent
-            %p Best Seller n.1
-        %div.gridContent
-            %p Best Seller n.1
+    %div.grid-container
+        - @categories.each do |category|
+            %div.grid-content{id: dom_id(category.products.first)}
+                %p= category.products.first.name
+                %p= category.products.first.description
+                %p= number_to_currency(category.products.first.price, unit: "kr", format: "%n %u")
+                %p= add_to_cart(category.products.first)  
```

Comment:
Why just first?  Why not iterate over `@category.products`?

Reply:
We are gonna use that feature on the product page. On the landing page we will just display one product from each category. I couldn't do it at this time because another group was working on the product page. Once this is merged we will put the loop on that page.

Reply:
Okay. I understand. Thank you for the clarification.